### ISSUE TITLE
Cannot find out the the image information file in firefox

### ISSUE THREAD

**Issue by dancorpse** [lovasoa/dezoomify#192]
I used to look for the actual image path( image information file ) by ABP for Firefox(show blocked items), it works perfectly! But unfortunately I just noticed that the ABP has updated and now become impossible to find out the wanted file, that is .xml, .dzi .json and etc. 
I've follow the wiki instruction https://github.com/lovasoa/dezoomify/wiki/Dezoomify-FAQ, but there's no such files in XHR section in the inspector.

**Comment by zstadler** [comment]
Alternatively, you can look at the "All" section, sort by type, and look the appropriate extensions.

**Comment by dancorpse** [comment]
Yes, I've tried that, but still cannot find out the desired file. Take following as an example: 
http://www.lukasweb.be/en/photo/epitaph-of-nikolaas-rockox-and-his-wife-adriana-perez-0,  
I know the correct xml file should be http://www.lukasweb.be/sites/default/files/zoomify/1698/a4533237e0bdca2c91ece72b637d63a3/ImageProperties.xml, but there's no files with .xml extension, no matter how I refresh the page. Can you help to have a try?

**Comment by lovasoa** [comment]
@dancorpse : I do not see any zoomable image in the page you give... Where is it ?

**Comment by dancorpse** [comment]
@lovasoa Sorry, here it is: http://www.lukasweb.be/en/zoom/1698 , the zoomable image is in a popup page.

**Comment by BartChris** [comment]
http://ophir.alwaysdata.net/dezoomify/dezoomify.html#http://www.lukasweb.be/sites/default/files/zoomify/1698/a4533237e0bdca2c91ece72b637d63a3/ImageProperties.xml
Here you go

**Comment by dancorpse** [comment]
@BartChris Oh, I'm so stupid, all I could find in "other" tab are many small jpeg tile files and some gif files as: zoominup.gif zoomindow.gif, and etc. I've refreshed the page many times to let the images loading completely...but still no .xml file.
